Exports don't have explicit types set

### Reproduction steps

1. Run `npx create-react-app sample-fail-browser-export --template typescript`
2. Run `npm install @trinsic/trinsic`
3. Add `import { AccountService } from "@trinsic/trinsic/lib/browser";` to your `App.tsx` file
4. Add `const service = new AccountService();` below imports in the `App.tsx` file
5. Run `npm start` and see error:

> ERROR in ./src/App.tsx 6:0-62
> Module not found: Error: Package path ./lib/browser is not exported from package /Users/jpgeorge/Development/temp/sample-fail-browser-export/node_modules/@trinsic/trinsic (see exports field in /Users/jpgeorge/Development/temp/sample-fail-browser-export/node_modules/@trinsic/trinsic/package.json)

### Reproduction Code

_No response_

--- a/web/package.json
+++ b/web/package.json
@@ -7,6 +7,16 @@
   },
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "typesVersions": {
+    "*": {
+        "browser": [
+            "./lib/browser/index.d.ts"
+        ],
+        "node": [
+            "./lib/node/index.d.ts"
+        ]
+    }
+  },
   "files": [
     "lib/**/*"
   ],

